coriolis/api/v1/views/endpoint_storage_view.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+import itertools
+
+
+def _format_storage(req, storage, keys=None):
+    def transform(key, value):
+        if keys and key not in keys:
+            return
+        yield (key, value)
+
+    return dict(itertools.chain.from_iterable(
+        transform(k, v) for k, v in storage.items()))
+
+
+def single(req, storage):
```

Comment:
Considering there is no `/endpoint/$ID/storage/$STORAGE_ID` API call (and probably never will) this method will never get used so please remove it.